### DISCUSSION TITLE
Self-hosted backend server with SQL/rDBMS support? (going to use it with Apache Superset as UI)

### DISCUSSION THREAD

**Discussion by greatcat19** [open-telemetry/community#1683] [Community Discussions]
Hello,

I'm going to use OpenTelemetry for app/web analytics. But I found most backend implementations are mainly about Cloud services.
For example, Jaeger is a mature solution but it is focused on the distributed system.
I only want something like https://posthog.com/ but I don't like their pricing strategy. 

Do we have any open source backend that can transform client OpenTelemetry data to SQL databases? It does not have to have a UI because I'm going to leverage Apache Superset.

**Comment by trask** [comment]
hi @greatcat19! i'd suggest asking in the [#opentelemetry](https://cloud-native.slack.com/archives/CJFCJHG4Q) slack channel

**Reply by greatcat19** [reply]
Sounds good! Thanks!
